src/main/java/com/google/devtools/build/lib/analysis/starlark/StarlarkRuleClassFunctions.java

```diff
@@ -304,7 +305,10 @@ public static RuleClass getTestBaseRule(RuleDefinitionEnvironment env) {
                         ExecutionTransitionFactory.createFactory(
                             DEFAULT_TEST_RUNNER_EXEC_GROUP_NAME))
                     .singleArtifact()
-                    .value(labelCache.get(toolsRepository + "//tools/test:collect_coverage")))
+                    .value(labelCache.get(toolsRepository + 
+                        (OS.getCurrent() == OS.WINDOWS ? 
+                            "//tools/test:collect_coverage_wrapper" : 
```

Comment:
It would be cleaner to keep the same label here and instead make it an `alias` with a `select` on `@platforms//os:windows` in the BUILD file.